torchgeo/datamodules/chesapeake.py

```diff
@@ -3,21 +3,53 @@
 
 """Chesapeake Bay High-Resolution Land Cover Project datamodule."""
 
-from typing import Any, Callable, Dict, List, Optional
+from typing import Any, Dict, List
 
-import torch
+import kornia.augmentation as K
+import torch.nn as nn
 import torch.nn.functional as F
-from pytorch_lightning.core.datamodule import LightningDataModule
+from einops import rearrange
 from torch import Tensor
-from torch.utils.data import DataLoader
-from torchvision.transforms import Compose
 
-from ..datasets import ChesapeakeCVPR, stack_samples
-from ..samplers.batch import RandomBatchGeoSampler
-from ..samplers.single import GridGeoSampler
+from ..datasets import ChesapeakeCVPR
+from ..samplers import GridGeoSampler, RandomBatchGeoSampler
+from ..transforms import AugmentationSequential
+from .geo import GeoDataModule
 
 
-class ChesapeakeCVPRDataModule(LightningDataModule):
+class _Transform(nn.Module):
```

Comment:
Should this have a TODO remove?

Reply:
Not sure how to remove. Kornia transforms turn all [B, C, H, W] and [C, H, W] into [B, C, H, W]. Personally I think that's a bug in Kornia (samples should not have a batch dimension added) but I'm not sure if I can convince the Kornia devs of this.